### ISSUE TITLE
initialValues from config and initializeForm bugs after v3.1.0

### ISSUE THREAD

**Issue by jsdmc** [erikras/redux-form#390]
Hello there.
I've faced a bug connected to initialization of form values using config property *initialValues* and *inializeForm* method
I briefly checked state of form and it seems that after calling *inializeForm* or setting default values through config form state looks like this .
```javascript```
{
   form: {
    myForm: {
      field1: { }
    }
  }
}
```
Later when setting form values all works as expected and form state takes values 
```javascript
{
   form: {
    myForm: {
      field1: {
        value: 2
      }
    }
  }
}
```

When using v3.1.0 and below it sets form state to this

```javascript
{
   form: {
    myForm: {
      field1: {
        initial: 1,
        value: 2
      }
    }
  }
}
```
Will provide a sample later

Cheers

**Comment by xcatliu** [comment]
Thx for reporting, same issue as https://github.com/erikras/redux-form/issues/360

**Comment by erikras** [comment]
This should be fixed in [`v4.0.0`](https://github.com/erikras/redux-form/releases/tag/v4.0.0). Come and get it!
